app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -142,8 +143,30 @@ class TabTrayView(
                 }
         }
 
-        fabView.new_tab_button.setOnClickListener {
-            interactor.onNewTabTapped(isPrivateModeSelected)
+        val accessibilityManager =
+            view.context.applicationContext.getSystemService(Context.ACCESSIBILITY_SERVICE) as? AccessibilityManager
+        val hasAccessibilityEnabled = accessibilityManager?.isEnabled ?: false
```

Comment:
@sblatz Is this block sufficient to know if the user is using talkback or another a11y service?  Anything more I should do or check?

Reply:
You should check against the settings param: `accessibilityServicesEnabled` it checks for both switch services and touch exploration services :)